awsmfa/__init__.py

```diff
@@ -273,10 +276,14 @@ def validate(args, config):
 
 
 def get_credentials(short_term_name, lt_key_id, lt_access_key, args, config):
-    console_input = prompter()
-    mfa_token = console_input('Enter AWS MFA code for device [%s] '
-                              '(renewing for %s seconds):' %
-                              (args.device, args.duration))
+    if args.token:
+        logger.debug("Received token as argument")
+        mfa_token = '%s' % (args.token)
```

Comment:
Is this to avoid the token being interpreted as an int?

Seems like adding `type=str` to the `add_argument` call could be a better fit with the rest of the code (`--duration` uses `type=int`).

Reply:
Added `type=str` in `add_argument`.